EIPS/eip-buy-russia-for-free-gas.md

```diff
@@ -0,0 +1,31 @@
+---
+eip: buy-russia-for-free-gas
+title: Buy Russia for free gas
+description: Ethereum Foundation should buy Russia to stop war on Ukraine and obtain virtually unlimited gas
+author: Nikita Kolmogorov (@backmeupplz)
+discussions-to: https://t.me/borodutcher
```

Comment:
Should be on Ethereum magicians

Reply:
Created a topic, but it has to be reviewed by a moderator. Will add the link to the Ethereum magicians as soon as the topic is up